cg/apps/bam_to_cram.py

```diff
@@ -0,0 +1,86 @@
+"""
+    Module for compressing bam to cram 
+"""
+
+import logging
+import tempfile
+from pathlib import Path
+
+from cg.utils import Process
+
+LOG = logging.getLogger(__name__)
+
+
+REFERENCE_PATH = "path/to/reference"
+SBATCH_SCRIPT_TEMPLATE = """
+#!/bin/bash
+#SBATCH -J {job_name}
```

Comment:
add sbatch parameters...

Reply:
resolution?

Reply:
Most of the parameters can be passed via the cg-config, or through the CLI. The --qos flag is however hardcoded to "low" for the sbatch scripts